template.go

```diff
@@ -26,14 +26,30 @@ type tplate struct {
 	contents     string
 	mode         os.FileMode
 	modeOverride bool
+
+	additionalTemplates templateAliases
 }
 
 func (t *tplate) toGoTemplate(g *gomplate) (*template.Template, error) {
 	tmpl := template.New(t.name)
 	tmpl.Option("missingkey=error")
 	tmpl.Funcs(g.funcMap)
 	tmpl.Delims(g.leftDelim, g.rightDelim)
-	return tmpl.Parse(t.contents)
+	tp, err := tmpl.Parse(t.contents)
+	if err != nil {
+		return tp, err
+	}
+	for alias, path := range t.additionalTemplates {
+		b, err := ioutil.ReadFile(path)
```

Comment:
you'll need to add `// nolint: gosec` above this line to get the linter to shut up:

```
template.go:43::warning: Potential file inclusion via variable,MEDIUM,HIGH (gosec)
```

It's more than a _potential_ file inclusion, it's a _deliberate_ file inclusion, but that's the whole point of this feature 😂